### ISSUE TITLE
Broken links in page "FINETUNING TORCHVISION MODELS" 

### ISSUE THREAD

**Issue by fabrizioschiano** [pytorch/pytorch.github.io#892]
## 📚 Documentation

I am going through the following documentation page and I have found several links that are broken: https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#finetuning-torchvision-models

**Some examples:**
1. in the paragraph https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#inputs there is a link to ImageFolder that is broken
2. in the paragraph https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#load-data when there is the word "here" in the sentence "[...] with the hard-coded normalization values, as described here."

There might be others. 

Thanks for the great tutorial.

**Comment by NeoKish** [comment]
The following link https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#finetuning-torchvision-models resides in the older version of the pytorch website. The same issue was being generated in the pytorch/tutorials issues segment https://github.com/pytorch/tutorials/issues/1008 mentioning that above tutorial is not available on the site/master branch. I am not sure why the tutorial has been removed but couldn't find in the latest version of the website.
